packages/http-client-python/eng/scripts/Test-Packages.ps1

```diff
@@ -34,7 +34,20 @@ try {
         Set-StrictMode -Version 1
         
         # run E2E Test for TypeSpec emitter
-        Write-Host "Generating test projects ..."
+        Write-Host "Generating test projects with pyodide ..."
+        & "$packageRoot/eng/scripts/Generate-WithPyodide.ps1"
+        Write-Host 'Code generation is completed.'
+
+        try {
+            Write-Host 'Checking for differences in generated code...'
+            & "$packageRoot/eng/scripts/Check-GitChanges.ps1"
+            Write-Host 'Done. No code generation differences detected.'
```

Comment:
have you made sure that this will catch differences in the generated code? You can try passing in an extra flag or something when generating with pyodide to make sure that any changes are caught